src/app/profile/page.tsx

```diff
@@ -7,27 +7,43 @@ import {
   DialogTitle,
   DialogActions,
   DialogContent,
+  Chip,
 } from '@mui/material'
 import { useDispatch } from 'react-redux'
-import { useState } from 'react'
+import React, { useState } from 'react'
 import { Button } from '@/components/elements/Button'
 import Link from 'next/link'
 import { useGetProfileQuery, useGetEventsQuery } from '@/redux/api/profileApi'
 import { logout } from '@/redux/features/userSlice'
+import { SubscriptionHistory } from './SubscriptionHistory'
+import { useGetSubscriptionsQuery } from '@/redux/api/subscriptionApi'
+
+type ChipType = 'event' | 'history'
 
 export default function Profile() {
   const [openPopup, setOpenPopup] = useState(false)
+  const [chipType, setChipType] = React.useState<ChipType>('event')
   const { data, isLoading, isError } = useGetProfileQuery()
   const { data: events } = useGetEventsQuery()
   const dispatch = useDispatch()
 
+  const { data: subscriptionHistory } = useGetSubscriptionsQuery()
+
+  const CHIP_STYLE = '!font-bold !p-5 !border-none'
```

Comment:
for constants value that doesnt change, please seperate it in to a constants.ts file. This is to avoid recreating the const every time the component is rendered